server/repository/src/migrations/views/mod.rs

```diff
@@ -102,12 +102,12 @@ pub(crate) fn rebuild_views(connection: &StorageConnection) -> anyhow::Result<()
         quantity_movement as quantity,
         item_id,
         store_id,
-        delivered_datetime as datetime
+        received_datetime as datetime
```

Comment:
Do you think we should have both `delivered` and `recieved` fields in the View?

Reply:
I think because it's stock movement, we only want the stock movement ones,which means it doesn't include delivered_datetime? Unless I'm missing something?